apps/asap-server/test/handlers/calendar/subscribe-handler.test.ts

```diff
@@ -42,11 +42,39 @@ describe('Calendar handler', () => {
   const handler = generateHandler(23, subscribe, unsubscribe, alerts);
 
   afterEach(() => {
-    subscribe.mockClear();
-    unsubscribe.mockClear();
+    subscribe.mockReset();
+    unsubscribe.mockReset();
+    calendarControllerMock.update.mockReset();
   });
 
   describe('Validation', () => {
+    test.each([
```

Comment:
to me using `.each` like this (ie for only two cases and with such complicated setup in the array) seems unnecessary - it always makes it a little harder to read

Reply:
@peterstarling yes I agree. I had more test cases before but deleted them after discussion with @kmaid. I'll remove the `.each`